This test fails because it requires libspa-test to be in the pipewire package.
We can probably build it and even split into its own package, but I don't think
it's necessary for now.

https://gitlab.freedesktop.org/pipewire/wireplumber/-/issues/18

diff --git a/tests/modules/meson.build b/tests/modules/meson.build
index 3e695dc..f3a2087 100644
--- a/tests/modules/meson.build
+++ b/tests/modules/meson.build
@@ -27,13 +27,6 @@ test(
   env: common_env,
 )
 
-test(
-  'test-si-node',
-  executable('test-si-node', 'si-node.c',
-      dependencies: common_deps, c_args: common_args),
-  env: common_env,
-)
-
 test(
   'test-si-audio-adapter',
   executable('test-si-audio-adapter', 'si-audio-adapter.c',
